NMS_Master/src/main/java/io/github/thatsmusic99/nms/FakePlayer.java

```diff
@@ -150,4 +169,14 @@ public void component(@NotNull String text) {
             result.append(text);
         }
     }
+
+    @Override
+    public boolean isOnPortalCooldown() {
+        return true; // Prevents mob from teleporting from a portal
```

Comment:
Can always change this in the future, I do want to emulate a player as much as possible with this.